samples/snippets/list_testcase_results_test.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+
+from list_testcase_results import list_test_case
+
+LOCATION = 'global'
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
```

Comment:
This relies on the same mechanism as Application Default Credentials to fetch the project from the environment: 

```suggestion
import google.auth

_, PROJECT_ID = google.auth.default()
```